src/cljs/athens/events.cljs

```diff
@@ -1574,13 +1592,12 @@
 
 (reg-event-fx
   :block/move
-  (fn [_ [_ {:keys [source-uid target-uid target-rel] :as args}]]
-    (log/debug ":block/move args" (pr-str args))
-    (let [atomic-event (common-events/build-atomic-event
-                         (atomic-graph-ops/make-block-move-op source-uid
-                                                              {:block/uid target-uid
-                                                               :relation target-rel}))]
-      {:fx [[:dispatch [:resolve-transact-forward atomic-event]]]})))
+  (fn [_ [_ {:keys [source-uid target-uid target-rel local-string] :as args}]]
+    (log/info ":block/move args" (pr-str args))
```

Comment:
nit: maybe keep it at `debug` level

Reply:
Urf durf, forgot this. Good catch! Fixed.